egs2/librispeech/asr1/conf/tuning/train_asr_conformer7_hubert_960hr_large.yaml

```diff
@@ -0,0 +1,89 @@
+# Trained with Tesla V100-SXM2(32GB) x 8 GPUs. It takes about 1.5 days.
+batch_type: numel
+batch_bins: 100000000
+accum_grad: 1
+max_epoch: 28
+patience: none
+init: none
+best_model_criterion:
+-   - valid
+    - acc
+    - max
+keep_nbest_models: 10
+
+freeze_param: [
+"frontend.upstream"
+]
+
+frontend_conf:
+  n_fft: 512
+  hop_length: 256
+
+frontend: s3prl
+frontend_conf:
+    frontend_conf:
+        upstream: hubert_large_ll60k  # Note: If the upstream is changed, please change the input_size in the preencoder.
+    download_dir: ./hub
+    multilayer_feature: True
+
+preencoder: linear
+preencoder_conf:
+    input_size: 1024  # Note: If the upstream is changed, please change this value accordingly.
```

Comment:
Should we also add a note to indicate the `input_size` for other models? Actually, the `input_size` is hard to find from s3prl.

Reply:
Good idea. I think we can mention it somewhere else. If we put it in this config, it's a bit confusing.